go-controller/pkg/ovn/gateway.go

```diff
@@ -26,10 +26,15 @@ const (
 	extSwitchToGwRouterPrefix    = "etor-"
 	gwRouterToExtSwitchPrefix    = "rtoe-"
 
-	nodeLocalSwitch          = "node_local_switch"
-	interNodePolicyPriority  = "1003"
-	nodeSubnetPolicyPriority = "1004"
-	mgmtPortPolicyPriority   = "1005"
+	nodeLocalSwitch = "node_local_switch"
+
+	// prority of logical router policies on the ovnClusterRouter
+	egressFirewallStartPriority = 10000
+	interNodePolicyPriority     = 1003
```

Comment:
i think we should keep the constants all in the same type here, either all string or all int

Reply:
made all the constants integers